src/app/admin/api-key/api-key-edit/api-key-edit.component.ts

```diff
@@ -30,6 +30,7 @@ export class ApiKeyEditComponent implements OnInit {
   public formFailedSubmit = false;
   public permissions: PermissionResponse[] = [];
   private organizationId: number;
+  id: number;
```

Comment:
This should be either private or public, compared to the rest of the properties :-)